### ISSUE TITLE
Milestone1 Issues

### ISSUE THREAD

**Issue by aimirza** [UBC-MDS/Song_Key_Mode_and_Popularity_on_Spotify#34]
**You may close this issue when all issues are resolved and checked.** 

**Mechanics**
- [ ] Name/title of README.md file should be in proper English. 
- [ ] **_(critical) No Milestone1 README.md. I only see the proposal README.md. See [instructions](https://github.ubc.ca/MDS-2018-19/DSCI_522_dsci-workflows_students/blob/master/release/milestone1/milestone1.md)._**
- [ ] Use propper citation format.
- [ ] Fix the folder named "results/figure" in your repo. It shouldnt have a slash. 
- [ ] list dependencies.

**Reasoning**
- [ ] Need more description of the data. For example, how was the data created? What is Spotify? How wsa the top 100 songs determined? 
- [ ] Whats the importance of your question? 
 [ ] Why did you choose an alpha of 0.1? That usualy very high. Did you mean 0.01? The most common is 0.05.
- [ ] In your main README.md file, explain how to run your data analysis (which order scripts are run in, what expected inputs are).

**Visualization**
- [ ] In the report, all tables need a number and description (e.g., Table 1. Estimates and confidence intervals), and all figures need a figure number and legend. 
- [ ] More descriptive Figure and table captions. Captions should stand alone. Meaning, the reader should understand the figure by examining the graph and caption alone, and not refer back to the text. For example: "Figure 1. Mean pregnancy length for first babies pregnancy and for other (non-first baby) pregnancy length. Error bars represent 95% confidence intervals generated by boostrapping.)"
- [ ] Table 1 is not a table. It looks like code comments. 
- [ ] (Figure on page 3) X-axis scale is wrong. Legend not readable. 

**Quality -- Readability and Documentation**
- [ ] Need more self-documentation for the script 02_data_exploration.R. Be more specific. "this script inputs the clean data frame `clean_top_tracks.csv` and gives out 2 visualizations" is not meaningful enough -- what are the 2 visualization? 
- [ ] Need more comments. It is hard to follow the scripts. 
- [ ] Instead of putting all or most of the codes under the main function, creat multiple functions and call those function under the main function. 
- [ ] Mention in which directory the script should be run. Example, in root. 
- [ ] The script 04_plot_testresults.R has comments at the beginning that seem misplaced and not relevent to the script. 

**Quality -- Robustness and Maintainability**
- [ ] Should have a unit test in each script (because we did not clearly mention this in the instructions, we did not reduct points for not having a unit test)

**Grammar and Spelling**
- [ ] 3 grammar or spelling mistakes.

**Comment by Pall-v** [comment]
Started tracking listed items via the original issue comment that lists them.

**Comment by aimirza** [comment]
I apologize, no need for a unit test because you don't have functions in your scripts other than the function main.

**Comment by Pall-v** [comment]
@aimirza do we need to create a new release for the last two points on this issue, or do the fixes just need to be a part of the repository?
